<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDoubleSpinBox" name="EpsilonSpinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_Epsilon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>391</width>
      <height>491</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditV">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="plainText">
     <string>function V(x1,x2)
   return( 
   0.5*x2^2-cos(x1)
   );
end		 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonV">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>use V</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditf_inf">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="plainText">
     <string>function f_inf(x1,x2)
   return( 
   x2-0.1;
   -sin(x1)-x2-0.1
   );
end		 </string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditf_sup">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>220</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="plainText">
     <string>function f_sup(x1,x2)
   return( 
   x2+0.1;
   -sin(x1)-x2+0.1
   );
end		 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonf_inf">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>use f_inf</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonf_sup">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>240</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>use f_sup</string>
    </property>
   </widget>
   <widget class="QSlider" name="vscrollbar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>20</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>v+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawV">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawV</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditgradV">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="plainText">
     <string>function gradV(x1,x2)
   return( 
   -sin(x1);
   x2
   );
end		 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtongradV">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>use gradV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawF">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawF</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="trajc">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>470</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>trajectory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSolve">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawVstable</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderResolution">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>16</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>450</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>resolution</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
